packages/kbn-es-archiver/src/actions/load.ts

```diff
@@ -94,8 +94,9 @@ export async function loadAction({
 
   await client.indices.refresh(
     {
-      index: '_all',
+      index: `_all,.kibana_${kibanaPackageJson.version}_001,.kibana_task_manager_${kibanaPackageJson.version}_001`,
```

Comment:
I think most archives go into `.kibana` which you're missing here...

As a quick hack maybe this would give better results:
```diff
--- a/packages/kbn-es-archiver/src/actions/load.ts
+++ b/packages/kbn-es-archiver/src/actions/load.ts
@@ -85,15 +85,17 @@ export async function loadAction({
   progress.deactivate();
   const result = stats.toJSON();
 
+  const indices = [];
   for (const [index, { docs }] of Object.entries(result)) {
     if (docs && docs.indexed > 0) {
       log.info('[%s] Indexed %d docs into %j', name, docs.indexed, index);
+      indices.push(index);
     }
   }
 
   await client.indices.refresh(
     {
-      index: '_all',
+      index: indices.join(','),
       allow_no_indices: true,
     },
     {
```